packages/graphback-cli/src/components/generate.ts

```diff
@@ -0,0 +1,271 @@
+import { CliPlugin } from "@test-graphql-cli/common";
+import { CodeFileLoader } from '@graphql-toolkit/code-file-loader';
+import { GraphbackCRUDGeneratorConfig } from "@graphback/core"
+import { migrateDB } from "graphql-migrations"
+import { GeneratedResolvers } from '@graphback/codegen-resolvers/types/api/resolverTypes';
+import Listr, { ListrTask } from 'listr';
+import { prompt } from 'inquirer';
+import chokidar from 'chokidar';
+import debounce from 'debounce';
+import { GraphQLExtensionDeclaration } from 'graphql-config';
+import { print } from 'graphql';
+
+export interface GenerateConfig {
+  folders: {
+    model: string;
+    schema: string;
+    resolvers: string;
+    client: string;
+    migrations: string;
+  };
+  graphqlCRUD: GraphbackCRUDGeneratorConfig;
+  generator: {
+    resolvers: ResolverGeneratorOptions
+    schema: SchemaGeneratorOptions
+    client: { format: 'ts' | 'gql' }
+  }
+  db: { database: string; dbConfig: any; };
+}
+
+const FormatExtensionMap = {
+  'ts': 'ts',
+  'js': 'js',
+  'gql': 'graphql',
+};
+
+export function writeFile(path: string, data: any) {
+  return new Promise<void>(async (resolve, reject) => {
+    await ensureFile(path);
+    fsWriteFile(path, data, err => {
+      if (err) {
+        reject(err);
+      }
+      resolve();
+    })
+  })
+}
+
+export function globPromise(glob: string, options: import('glob').IOptions = {}) {
```

Comment:
import glob at top of file is preferred.

Reply:
Why import glob at function level (and twice)?

Reply:
This is kinda strange codestyle that is all over the CLi. I will replace that soon

Reply:
I've seen this happen when using VS Code's extract function.